diesel_tests/tests/schema/mod.rs

```diff
@@ -260,9 +260,9 @@ pub fn backend_specific_connection() -> TestConnection {
     // we do match the error messages in some tests and depending on your
     // operating system configuration postgres may return localized error messages
     // This forces the language to english
-    diesel::sql_query("SET lc_messages TO 'en_US.UTF-8'")
-        .execute(&mut conn)
-        .unwrap();
+    /*diesel::sql_query("SET lc_messages TO 'en_US.UTF-8'")
+    .execute(&mut conn)
+    .unwrap();*/
```

Comment:
Is there any particular reason to comment out this code?

Reply:
I did it because I used a database from my employer to test this functionality (https://neon.tech). However, some features are restricted, from what I understand security reasons (and we already default to the `C.UTF-8` locale).

In any case, this was never supposed to have made it to github, so I've removed it again from the PR.